wandb/sdk/__init__.py

```diff
@@ -16,4 +16,14 @@
 from .wandb_settings import Settings  # noqa: F401
 from .wandb_setup import setup  # noqa: F401
 from .wandb_summary import Summary  # noqa: F401
+from .wandb_verify import (  # noqa: F401
+    check_artifacts,
+    check_graphql_put,
+    check_host,
+    check_large_post,
+    check_logged_in,
+    check_run,
+    check_secure_requests,
+    check_wandb_version,
+)
```

Comment:
These shouldnt be there, they are not public functions.

Reply:
Got it